web-client/src/presenter/sequences/gotoDashboardSequence.js

```diff
@@ -59,6 +59,7 @@ const goToDashboard = [
                     USER_ROLES.clerkOfCourt,
                     USER_ROLES.docketClerk,
                     USER_ROLES.floater,
+                    USER_ROLES.petitioner,
```

Comment:
Does this addition mean that petitioner roles will run the `proceedToMessages` action list?

Reply:
Removed petitioner.  I misread what it was doing. I made this change initially because in the integration test, there was a failure which I thought I had traced back to this.  I just forgot to remove it once I saw that this was not the cause.  Thanks for catching it.